services/web/server/src/simcore_service_webserver/projects/_projects_db.py

```diff
@@ -35,6 +36,15 @@
     projects.c.hidden,
     projects.c.workspace_id,
     projects.c.trashed,
+    projects.c.trashed_by,  # == user.id (who trashed)
```

Comment:
this not a group_id? also Sylvain commented something about it being wrong in the description above

Reply:
From the title of the PR it looks like you are using  the user's `gid` instead of it `id`

Reply:
Sorry, I did a mistake in the description. 

It is GROUPID in the API and USERID in the DB